package.json

```diff
@@ -16,7 +16,7 @@
     "db:seed": "sequelize db:seed:all",
     "start": "node dist/index.js",
     "start:dev": "DEBUG=dev nodemon --exec babel-node server",
-    "test": "nyc mocha --timeout 1000 -r esm --exit"
+    "test": "npm run db:rollback && nyc mocha --timeout 1000 -r esm --exit"
```

Comment:
I wonder why we still have this again here

Reply:
It is for running migrations before the test runs. If we don't have it then our tests will fail on Travis because there is no migration to run. Remember I had to remove it from the bug fix branch because I also removed the migrations and models.